include/tvm/expr_operator.h

```diff
@@ -700,6 +700,9 @@ inline Expr make_zero(Type t) {
   }                                                            \
   inline Expr Name(const Expr& a, int b) {                     \
     return Name(a, make_const(a.type(), b));                   \
+  }                                                            \
+  inline Expr Name(const Expr& a, double b) {                  \
+    return Name(a, make_const(Float(64), b));                  \
```

Comment:
Why do you need to use a double instead of float? This would be one of the only places in the codebase where doubles are used.

Reply:
I want to make it consistent with other attrs as in include/tvm/relay/attrs/nn.h, all float values are defined as double.